backend/bin/wait-for-db.php

```diff
@@ -6,12 +6,12 @@
 
 $connected = false;
 while (!$connected) {
-    $app = \eCampApp::CreateSetup();
-    $sm = $app->getServiceManager();
-    /** @var \Doctrine\ORM\EntityManager $em */
-    $em = $sm->get('doctrine.entitymanager.orm_default');
-    $conn = $em->getConnection();
     try {
+        $app = \eCampApp::CreateSetup();
+        $sm = $app->getServiceManager();
+        /** @var \Doctrine\ORM\EntityManager $em */
+        $em = $sm->get('doctrine.entitymanager.orm_default');
+        $conn = $em->getConnection();
```

Comment:
Durch diese Verschiebung wird das Skript ewig loopen wenn z.B. im Service Manager eine Exception auftritt. Das ist mir beim Testen des PRs sogar einmal passiert, und ich war zunächst durch die resultierenden Fehlermeldungen verwirrt:
```
Waiting for DB to come online...
blah blah blah something with Textarea Module could not be initialized
Waiting for DB to come online...
blah blah blah something with Textarea Module could not be initialized
Waiting for DB to come online...
blah blah blah something with Textarea Module could not be initialized
...
```
Ohne die Änderung wäre der Backend-Container in diesem Fall einfach beendet worden, und hätte die Stack Trace einmalig ausgespuckt, ohne die falsche Nachricht über die DB.

Mit deiner Änderung ist es dafür möglich, eine falsche Datenbank-Konfiguration noch während dem Startvorgang manuell anzupassen... Weiss aber nicht ob das oft genug der Fall ist um die Änderung rechtzufertigen, wenn auf allen OS Docker eingesetzt werden kann und die DB-Config somit standardisiert ist.

Reply:
Das Problem ist, dass Zeile 10 `\eCampApp::CreateSetup()` bereits zur DB connected und auf die Schnauze fliegt, wenn keine DB da ist. Und dann bricht `wait-for-db.php` munter ab und der Container rennt weiter, auch ohne DB.

Reply:
Change "Waiting for DB" to "Could not yet connect to DB"